src/components/HomeNav.js

```diff
@@ -40,38 +39,4 @@ class HomeNav extends Component {
   }
 }
 
-const HomeNavMenuDropdownButton = React.createClass({
-  getInitialState () {
-    return {
-      isOpen: false
-    }
-  },
-
-  toggleMenu (event) {
-    // event.stopPropagation()
-    this.setState({isOpen: !this.state.isOpen})
-  },
-
-  onClose () {
-    this.setState({isOpen: false})
-  },
-
-  componentDidMount () {
-    document.body.addEventListener('click', this.onClose)
-  },
-
-  componentWillUnmount () {
-    document.body.removeEventListener('click', this.onClose)
-  },
-
-  render () {
-    return <div>
-      <a onClick={this.toggleMenu} href='#'>More
-          <div className='more-triangle' />
-      </a>
-      <HomeNavDropdownMenu isOpen={this.state.isOpen} />
-    </div>
-  }
-})
-
 export default HomeNav
```

Comment:
JRob has requested that "export default" is included inline with the declaration of the component.